From 0fdc68eb96b2820eedbe789c2544b2bc3a49c549 Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Tue, 1 Nov 2011 12:13:57 -0700
Subject: [PATCH] inst_script should call inst_simple (#750603)

inst_script checks for a shebang, if it doesn't exist it exits.
If it does it should not be calling inst_binary, it should call
inst_simple like it used to.
---
 dracut-functions |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/dracut-functions b/dracut-functions
index 23aa14f..5e2acd3 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -449,7 +449,7 @@ inst_script() {
     [[ $debug ]] && _line=$(echo -n "$_line" | tr -c -d '[:print:][:space:]')
     _shebang_regex='(#! *)(/[^ ]+).*'
     [[ $_line =~ $_shebang_regex ]] || return 1
-    inst "${BASH_REMATCH[2]}" && inst_binary "$@"
+    inst "${BASH_REMATCH[2]}" && inst_simple "$@"
 }
 
 # same as above, but specialized for symlinks
-- 
1.7.6.4

